### PR TITLE
Registration App - Update "Add User" (FE)

### PR DESCRIPTION
**Pull request by vsamarora**
Registration App - Update "Add User" (FE)

### CODE REVIEW

File: portal/code/company-portal/src/components/cax-responsibilities.tsx [hunk 1]
```diff
@@ -95,11 +95,32 @@ export const ResponsibilitiesCax = ({
       };
 
       addToInviteList(data);
+      
+      var apiData = [];
+      apiData.push(data);
+
+      const fetchData = async () => {
+    const dataRoles = await submitSendInvites(apiData);
```
Comment (oyo) [inline portal/code/company-portal/src/components/cax-responsibilities.tsx]: maybe format the code :)

File: portal/code/company-portal/src/components/cax-responsibilities.tsx [hunk 2]
```diff
@@ -95,11 +95,32 @@ export const ResponsibilitiesCax = ({
...
+            toast.success(dataRoles);
+      }
+      fetchData()
+      .catch((errorCode: number) => {
+      
+        let message = DataErrorCodes.includes(errorCode)
```
Comment (oyo) [inline portal/code/company-portal/src/components/cax-responsibilities.tsx]: better use const when the variable doesn't change

File: portal/code/company-portal/src/components/cax-responsibilities.tsx [hunk 3]
```diff
@@ -242,13 +211,46 @@ export const ResponsibilitiesCax = ({
...
+                      </li>
+                    );
+                  })}
+                </ul>
+              </Row>
+            </Row>
+          )}
+
+          {/* <Row className="mx-auto col-9">
+            <h5>{t("Responsibility.userInvite")}</h5>
```
Comment (oyo) [inline portal/code/company-portal/src/components/cax-responsibilities.tsx]: remove this?

**Review by oyo — APPROVED** [review]
approved - but check some minor comments

File: portal/code/company-portal/src/components/cax-responsibilities.tsx (hunk 1 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/cax-responsibilities.tsx]: Updated

File: portal/code/company-portal/src/components/cax-responsibilities.tsx (hunk 2 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/cax-responsibilities.tsx]: updated

File: portal/code/company-portal/src/components/cax-responsibilities.tsx (hunk 3 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/cax-responsibilities.tsx]: updated

**Pull request merged**